frontend/src/components/content-grid/preview/generic.vue

```diff
@@ -39,5 +51,15 @@ c-anchor-root.group.w-full(class="max-w-[416px]")
         | {{ viewership }} {{ viewershipName + (viewership === 1 ? '' : 's') }}
         | •
         | {{ dateToElapsedTimeString(beginAt) }}
-    c-icon.text-slate-300(v-if="icon" :name="icon")
+    .flex.flex-col.justify-between.items-center
+      template(v-if="kebabMenu")
+        c-dropdown.flex.justify-center(class="[&_>_ul]:w-fit [&_>_ul]:right-0 [&_>_ul]:top-full")
+          c-dropdown-item(class="[&_>_li]:!bg-frost-blue hover:[&_>_li]:!bg-frost-deep")
```

Comment:
I think we should just make the button purple instead of making everything blue, as it's simpler and more consistent. Even if we did decide to introduce a new colour, it should be done through props as mentioned in a comment below.